src/components/screens/Service.tsx

```diff
@@ -76,10 +76,11 @@ export const Service = ({
     (item: TServiceTile, index: number) => (
       <ServiceTile
         draggableId={umlautSwitcher(item.title) || umlautSwitcher(item.accessibilityLabel)}
+        draggableKey={`item${item.title || item.accessibilityLabel}-index${index}`}
         hasDiagonalGradientBackground={hasDiagonalGradientBackground}
         isEditMode={isEditMode}
         item={item}
-        key={index + (item.title || item.accessibilityLabel)}
+        key={`item${item.title || item.accessibilityLabel}-index${index}`}
```

Comment:
why do we need both keys?

Reply:
Because we really use it for the key property, it is not a specialized prop. We cannot catch the key prop with `child.props.key` that we use in the `DraggableItem` component in `DraggableGrid`. Instead of key prop, we need a key with the same value as key (not necessarily the same, but uniq) but with a different prop name. This way we can catch the key for `DraggableItem` and use it. Otherwise `child.props.key` is always `undefined`.